mmr-backend/library.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+from enum import Enum
+
+
+class Library:
+    def __init__(self, owner, name: str, entries: list[Library.Entry] = list()):
+        self.__owner = owner
+        self.__name: str = name
+        self.__entries: list[Library.Entry] = entries
+
+    class ReadingStatus(Enum):
+        PLAN_TO_READ = "PLAN TO READ"
+        CURRENTLY_READING = "CURRENTLY READING"
+        COMPLETED = "COMPLETED"
+        DROPPED = "DROPPED"
+        ON_HOLD = "ON HOLD"
+
+    def to_dict(self):
```

Comment:
If this is an interface you're going to use throughout your classes, you might want to define an abstract base class to settle it down.

Reply:
We have a rule at work, which I think it's mentioned somewhere in the Clean Code book (not sure though), that says you can repeat code two times and that's fine, but when you get to do the third one then it's time to do some refactor.

If I get to need it a third time, which I'm not 100% sure will happen at this point, I'll just have to spend 10 minutes refactoring, I'll worry about it in the future